java/code/src/com/redhat/rhn/domain/server/ServerFactory.java

```diff
@@ -115,10 +107,11 @@ protected static CustomDataValue getCustomDataValue(CustomDataKey key,
 
         Session session = HibernateFactory.getSession();
         return (CustomDataValue) session.getNamedQuery(
-                "CustomDataValue.findByServerAndKey").setParameter("server",
-                        server).setParameter("key", key)
-                        // Retrieve from cache if there
-                        .setCacheable(true).uniqueResult();
+                "CustomDataValue.findByServerAndKey")
```

Comment:
```suggestion
                return (CustomDataValue) session.getNamedQuery("CustomDataValue.findByServerAndKey")
```
Getting the query name with the function call is likely to be less than 120, and the alignment is confusing.